tools/conformance.py

```diff
@@ -557,12 +564,16 @@ def __init__(self, name, path, options, os):
         elif self.os.is_mac():
             if self.name == 'chrome' or self.name == 'chrome_stable':
                 self.path = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
-            if self.name == 'chrome_canary':
+            elif self.name == 'chrome_canary':
                 self.path = '/Applications/Google Chrome Canary.app/Contents/MacOS/Google Chrome Canary'
+            elif self.name == 'safari':
+                self.path = '/usr/bin/safaridriver'
```

Comment:
This should be browser binary path, instead of webdriver path.

Reply:
Not for Safari. You don't worry about the browser path - instead you talk to the `safaridriver` binary and it will handle the browser for you. There is a strong mapping between a built `safaridriver` and the browser. (with the exception of iOS, where any `safaridriver` will forward requests onto the hidden driver running on the device)

Reply:
In fact, this could be left empty for Safari since it doesn't need to be used.